example/exporter/kubelet-servicemonitor.yaml

```diff
@@ -0,0 +1,20 @@
+# You need to have "--kubelet-object=kube-system/kubelet" as a prometheus-operator arg
```

Comment:
The `ServiceMonitor` is still required as that feature only synchronizes the `InternalIP`s of your `Node`s into a `Service`/`Endpoints` object. How you organize the `ServiceMonitor`s is still up to you. More elaborate cluster monitoring guide coming in #188 .